docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -563,6 +563,14 @@ directory.
 * - `iceberg.jdbc-catalog.schema-version`
   - JDBC catalog schema version.
     Valid values are `V0` or `V1`. Defaults to `V1`.
+* - `iceberg.jdbc-catalog.retryable-status-codes`
```

Comment:
> Add iceberg.jdbc-catalog.retryable-status-codes configuration property.
 
Please follow the commit message guideline: https://trino.io/development/process#contribution-process
> Do not end the subject line with a period.

Reply:
Removed the trailing period.

Reply:
I mean a commit title, not PR title.

Reply:
Updated commit message as well.